src/main/java/frc/robot/commands/Drive.java

```diff
@@ -51,6 +53,10 @@ public void execute() {
     yVelocity = conDriver.getAxisLSY();
     rVelocity = conDriver.getAxisRSX();
 
+    xVelocity = MathUtil.applyDeadband(xVelocity, constControllers.DRIVER_LEFT_STICK_X_DEADBAND);
+    yVelocity = MathUtil.applyDeadband(yVelocity, constControllers.DRIVER_LEFT_STICK_Y_DEADBAND);
+    rVelocity = MathUtil.applyDeadband(rVelocity, constControllers.DRIVER_RIGHT_STICK_X_DEADBAND);
+
```

Comment:
We should probably add this config to SuperCORE controller. I can see us forgetting to add this to a command when it really should be applied to the entire joystick right?